pkg/report/linux.go

```diff
@@ -484,10 +485,8 @@ func symbolizeLine(symbFunc func(bin string, pc uint64) ([]symbolizer.Frame, err
 	}
 	var symbolized []byte
 	for _, frame := range frames {
-		file := frame.File
-		file = strings.TrimPrefix(file, strip)
-		file = strings.TrimLeft(file, "./")
-		info := fmt.Sprintf(" %v:%v", file, frame.Line)
+		path, _ := backend.CleanPath(frame.File, ctx.kernelObj, ctx.kernelSrc, ctx.kernelBuildSrc, nil)
```

Comment:
Can you please ensure that syzkaller compiles between every pair of your patches?
This patch depends on backend.CleanPath(), which is only introduced by the following patch, so the two patches need to be reordered.

Reply:
The idea is to merge https://github.com/google/syzkaller/pull/5002 first, then https://github.com/google/syzkaller/pull/4985 and then this PR.

Will get these approved first.